framework/include/controls/RealFunctionControl.h

```diff
@@ -62,7 +62,7 @@ class RealFunctionControl : public Control
   Function & _function;
 
   /// Vector of parameters to change
-  std::vector<Real *> _parameters;
+  ControllableParameter<Real> _parameters;
```

Comment:
Well - this really has more to do with my comment that `ControllableParameter` shouldn't even exist... and you shouldn't `get` them at all.